src/topology/metric_space/gromov_hausdorff.lean

```diff
@@ -639,9 +639,9 @@ begin
       assume x,
       have : x ∈ ⋃y∈(s q), ball y ε := (hs q).2 (mem_univ _),
       rcases mem_bUnion_iff.1 this with ⟨y, ys, hy⟩,
-      let i := ((E q) ⟨y, ys⟩).1,
-      let hi := ((E q) ⟨y, ys⟩).2,
-      have ihi_eq : (⟨i, hi⟩ : fin (N q)) = (E q) ⟨y, ys⟩, by rw fin.ext_iff,
+      let i : ℕ := ((E q) ⟨y, ys⟩),
```

Comment:
```suggestion
      let i : ℕ := (E q) ⟨y, ys⟩,
```
Lots of extra parentheses in this section.

Reply:
Good catch. Fixed.